dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/Standard/HVDCStandard.mo

```diff
@@ -0,0 +1,160 @@
+within Dynawo.Electrical.HVDC.Standard;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model HVDCStandard "HVDC Standard model"
+
+  import Modelica;
+  import Dynawo;
+  import Dynawo.Types;
+  import Dynawo.Connectors;
+  import Dynawo.Electrical.SystemBase;
+
+  Dynawo.Connectors.ACPower terminal1(V(re(start = u10Pu.re), im(start = u10Pu.im)), i(re(start = i10Pu.re), im(start = i10Pu.im))) annotation(
```

Comment:
I see the connectors very big. @adriguir doesn't. What do you see @RomainLosseauRTE ?

Reply:
Same on my OM, terminals appear five times bigger than other blocks

Reply:
I corrected it. I just had a different setting in OMEdit (400x400 somewhere instead of 100x100). As a result, I had a 40x40 terminal instead of a 10x10 one. Normally you should now be able to see it with the correct size.